espnet2/gan_svs/hifigan/__init__.py

```diff
@@ -0,0 +1,17 @@
+from espnet2.gan_tts.hifigan.hifigan import (
```

Comment:
I think you can directly use the one from TTS by directly import them, so no need to re-define them here